Frameworks/Foundation/NSOrderedSet.mm

```diff
@@ -16,44 +16,41 @@
 
 #include "Starboard.h"
 #import <StubReturn.h>
+#include <vector>
 
-#include "Foundation/NSOrderedSet.h"
+#import <Foundation/NSOrderedSet.h>
+#import <Foundation/NSException.h>
+
+@implementation NSOrderedSet {
+    std::vector<StrongId<id>> setContainer;
```

Comment:
usage of set would lead to the order being different. when it removes duplicates, the last encountered duplicate ends up coming at the start of the iterator. so the order is not maintained.